README.md

```diff
@@ -2,6 +2,16 @@
 
 Drop it allows one click inserts/uploads of photos among the thousands of photos available on <a href="https://unsplash.com">unsplash.com</a> without leaving the Gutenberg editor.
 
+## How to
+
+After installing please run the following commands to make it work : 
+
+`npm install && npm run build`
+
+Please follow this tutorial https://youknowriad.github.io/gcf/#/contribute if you want to contribute.
+
+This plugin is available on wordpress.org if you just want to enjoy it without having any command line to run : https://wordpress.org/plugins/dropit/
```

Comment:
Can we move this to a specific section above the "How To"

`## Install`

Reply:
done ;)

Reply:
forgot that part ahah sorry. It should be good now.